src/main/java/org/jenkinsci/test/acceptance/docker/Docker.java

```diff
@@ -158,6 +122,80 @@ public DockerImage build(Class<? extends DockerContainer> fixture) throws IOExce
         }
     }
 
+    //package scope for testing purposes. Ideally we should encapsulate Docker interactions so they can be mocked
+    // and call public method.
+    void copyDockerfileDirectory(Class<? extends DockerContainer> fixture, DockerFixture f, File dir)
```

Comment:
Ok I have seen that in this project it is using java8 annotations for non nullable.